testing/jormungandr-testing-utils/resources/explorer/graphql/schema.graphql

```diff
@@ -29,36 +32,35 @@ type Block {
 }
 
 type BlockConnection {
+  """Information to aid in pagination."""
   pageInfo: PageInfo!
-  edges: [BlockEdge!]!
 
-  """
-  A count of the total number of objects in this connection, ignoring pagination.
-  """
-  totalCount: BlockCount!
+  """A list of edges."""
+  edges: [BlockEdge]
+  totalCount: Int!
 }
 
-scalar BlockCount
-
 """Block's date, composed of an Epoch and a Slot"""
 type BlockDate {
   epoch: Epoch!
   slot: Slot!
 }
 
+"""An edge in a connection."""
 type BlockEdge {
+  """The item at the end of the edge"""
   node: Block!
 
   """A cursor for use in pagination"""
-  cursor: IndexCursor!
+  cursor: String!
```

Comment:
I guess you have to decustomize `IndexCursor` for _reasons_, even though it's present in the queue parameters.

Reply:
Hmm, this is weird, actually... So it seems `async-graphql` let's you use a particular type as a cursor, but it puts `String` in the schema regardless... Ok, I guess I'll just get rid of `IndexCursor` altogether.